projects/ag-grid/documentation/code-examples/data-grid/data-manager-added/data-manager-data-grid-docs-demo.component.ts

```diff
@@ -0,0 +1,376 @@
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  OnInit,
+} from '@angular/core';
+import { SkyCellType, SkyAgGridService } from '@skyux/ag-grid';
+import {
+  SkyDataManagerService,
+  SkyDataManagerState,
+  SkyDataViewConfig,
+} from '@skyux/data-manager';
+
+import {
+  GridApi,
+  GridReadyEvent,
+  GridOptions,
+  ValueFormatterParams,
+  ColumnApi,
+  ColDef,
+} from 'ag-grid-community';
+
+import { SKY_AG_GRID_DEMO_DATA } from './data-manager-data-grid-docs-demo-data';
+import { DataManagerDataGridDocsDemoFiltersModalComponent } from './data-manager-data-grid-docs-demo-filter-modal.component';
+
+@Component({
+  selector: 'app-data-manager-data-grid-docs-demo',
+  templateUrl: './data-manager-data-grid-docs-demo.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class SkyDataManagerDataGridDemoComponent implements OnInit {
+  public columnDefs: ColDef[] = [
+    {
+      field: 'selected',
+      type: SkyCellType.RowSelector,
+    },
+    {
+      field: 'name',
+      headerName: 'Name',
+    },
+    {
+      field: 'age',
+      headerName: 'Age',
+      type: SkyCellType.Number,
+      maxWidth: 60,
+    },
+    {
+      field: 'startDate',
+      headerName: 'Start Date',
```

Comment:
Sentence-style capitalization...
```suggestion
      headerName: 'Start date',
```